examples/ftp-client/ftp_client.bal

```diff
@@ -35,6 +35,4 @@ public function main() returns error? {
     // Closes the file stream to finish the `get` and `put` operations.    
     check fileStream.close();
 
```

Comment:
Extra empty line? Check other places as well.

Reply:
This extra empty line is used to avoid highlighting the line 38 (i.e. `}`) when the line 36 is has the hover event in the BBE site.